### PR TITLE
Add tabs in login - separate for system and non-system logins

### PR DESCRIPTION
**Pull request by krritik**
Separate source logins into two tabs - system and non-system logins. Since non-system logins cannot be migrated.
Created separate tabs for both.
image
image

### CODE REVIEW

File: extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts [hunk 1]
```diff
@@ -344,10 +345,29 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 			await this.populateLocationDropdown();
 
 			// Collect source login info here, as it will speed up loading the next page
-			const sourceLogins: LoginTableInfo[] = [];
+			var sourceLogins: LoginTableInfo[] = [];
```
Comment (Ramudaykumar) [inline extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts:348]: why are we changing this from const to var?

**Review by Ramudaykumar — APPROVED** [review]
Looks good to me, hope you have tested functionality as per the requirements. Approving it with couple comments.

**Comment by abhims14** [comment]
Text for system login tab : "Excluded login/s"
Info box for Excluded login/s text: "Below login/s is/are excluded from login migration as they are either local service or system accounts at the source."

File: extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts (hunk 1 above)
Comment (krritik) [reply extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts:348]: Previously we were not modifying its contents. But now we remove system logins from sourceLogins in a separate vairable.

**Comment by krritik** [comment]
> Looks good to me, hope you have tested functionality as per the requirements. Approving it with couple comments.

Yes @Ramudaykumar, I have verified it.

**Comment by krritik** [comment]
> Text for system login tab : "Excluded login/s" Info box for Excluded login/s text: "Below login/s is/are excluded from login migration as they are either local service or system accounts at the source."

@abhims14, I did the following changes. Can you verify same in the screenshots attached above?

**Comment by kburtram** [comment]
@krritik are you planning to complete this PR?  Or should we close it as abandoned?

**Comment by krritik** [comment]
@kburtram, we are planning to complete it in few days. So for now lets keep it open.

**Pull request closed without merging**